crates/humanode-runtime/src/tests/fixed_supply.rs

```diff
@@ -73,14 +119,52 @@ fn new_test_ext_with() -> sp_io::TestExternalities {
     storage.into()
 }
 
+fn assert_total_issuance() {
+    // Build the state from the config.
+    new_test_ext_with().execute_with(move || {
```

Comment:
This executes within it's own runtime state, so it does not test anything

Reply:
What do you not test anything. If I change formulas to be checked then I get errors in case the formulas are wrong.

Reply:
Got it. In that case, just drop the `new_test_ext_with().execute_with(move || {` `})` wrapping from the test.

Reply:
Thanks, my bad. Fixed.